ddtrace/mocktracer/mocktracer.go

```diff
@@ -40,13 +41,33 @@ type Tracer interface {
 // to activate the mock tracer. When your test runs, use the returned
 // interface to query the tracer's state.
 func Start() Tracer {
+	return StartWithOptions()
+}
+
+// StartWithOptions is the same as Start, but it allows to specify options.
+func StartWithOptions(opts ...StartOption) Tracer {
 	var t mocktracer
+	for _, o := range opts {
+		o(&t)
+	}
 	internal.SetGlobalTracer(&t)
 	internal.Testing = true
 	return &t
 }
 
+// StartOption is an option type to be passed to StartWithOptions
+type StartOption func(*mocktracer)
+
+// WithServiceName specifies the default service name
+func WithServiceName(serviceName string) StartOption {
```

Comment:
The parameter here should be `name` for consistency with other `WithServiceName` functions.